### PR TITLE
Fix processing error

### PR DESCRIPTION
**Pull request by rjschwei**
- Handle format change of repo config file
- Signal critical error if we cannot parse the config file

### CODE REVIEW

File: check_rmt_repos [hunk 1]
```diff
@@ -148,36 +148,42 @@ not_enabled = {'ids': [], 'descriptions': []}
 missing_repo_dirs = []
 extra_enabled_repos = {'ids': [], 'descriptions': []}
 extra_mirrored_repos = []
-with open(args.config_file_path) as json_file:
+config_repos = []
+try:
+    json_file = open(args.config_file_path)
```
Comment (jesusbv) [inline check_rmt_repos:153]: can we use `with open(args.config_file_path):` ?

Comment (rjschwei) [reply check_rmt_repos:153]: If you can explain the technical advantage of `with` vs the current approach sure.

At present the file handle goes out of scope after the try block and will get garbage collected whenever the interpreter cares to. The file is not other wise used and therefore the timing of the file handle going away is immaterial.

Comment (jesusbv) [reply check_rmt_repos:153]: I mentioned it to keep the implementation as it was (with the `with open` statement) as I understand the point of the change is the try/except block

File: check_rmt_repos [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright (c) 2019 SUSE LLC  All rights reserved.
+# Copyright (c) 2021 SUSE LLC  All rights reserved.
```
Comment (jesusbv) [inline check_rmt_repos]: s/2021/2022 ?

**Review by jesusbv — APPROVED** [review]
LGTM

**Pull request merged**